dora/core/client/fs/src/test/java/alluxio/client/file/ufs/AbstractUfsStreamTest.java

```diff
@@ -78,9 +57,10 @@ public void before() {
     mRootUfs = new AlluxioURI(ufs);
     UnderFileSystemFactoryRegistry.register(new LocalUnderFileSystemFactory());
     final FileSystemOptions fileSystemOptions =
-        FileSystemOptions.Builder
-            .fromConf(mConf)
+        new FileSystemOptions.Builder()
             .setUfsFileSystemOptions(new UfsFileSystemOptions(ufs))
+            .setDoraCacheEnabled(false)
+            .setDataCacheEnabled(false)
```

Comment:
fyi @dbw9580 @JiamingMai @LuQQiu  In the test for UfsFileStream we are manually disabling meta/data client cache. Let us know if this is incorrect. Thanks!

Reply:
looks good to me